.github/CODEOWNERS

```diff
@@ -22,9 +22,13 @@
 /sdk/ @lmazuel
 
 # Core
+# AzureSdkOwners: @xiangyan99 @kashifkhan @pvaneck
```

Comment:
Because this block is tied to a source path, the owners on that source path are considered the "service owners" if "Service Attention" is applied.    I'd suggest breaking this into PR and Issue blocks for clarity.

Same for other Azure SDK libraries with no service backing them (and thus no service owner).    

```text
# Core
# PRLabel: %Core.Http
/sdk/core/corehttp/       @pvaneck @lmazuel @annatisch @johanste @xiangyan99 @iscai-msft @kashifkhan

# ServiceLabel: %Core.Http
# AzureSdkOwners: @xiangyan99 @kashifkhan @pvaneck
```

The problem with this at the moment is the linter complains because it requires a service owner for all service labels, even when that doesn't make sense.   I've got a question out to Jim for the recommended way to work around that or a request to loosen that validation.

Reply:
Do we have ETA when the work around will be ready. Now it blocks the CI.

Reply:
No ETA yet.   Jim is looking at it.   Short-term mitigation would be to add a `ServiceOwners` line with the same Azure SDK team contacts.   I'm leaving mine open for a couple of days to see if there's any official fix coming.

Reply:
@xiangyan99 :  The linter has been updated.   Closing/reopening the PR to kick off a new run with the latest config should work for these blocks.

Reply:
Updated.